kotlin/src/main/kotlin/io/github/xanthic/cache/ktx/CacheApiExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.xanthic.cache.ktx
+
+import io.github.xanthic.cache.api.Cache
+import io.github.xanthic.cache.core.CacheApi
+import io.github.xanthic.cache.core.CacheApiSpec
+import java.util.function.Consumer
+
+/**
+ * @see CacheApi.create
+ */
+fun <K, V> createCache(init: CacheApiSpec<K, V>.() -> Unit): Cache<K, V> = CacheApi.create(Consumer(init))
```

Comment:
aside: I'd like this extension to be statically defined on `CacheApi` itself, but am blocked by https://youtrack.jetbrains.com/issue/KT-11968